.github/workflows/docs.yml

```diff
@@ -0,0 +1,43 @@
+name: Deploy documentation
+
+on:
+  push:
+    branches:
+      - main
+
+jobs:
+  changes:
+    runs-on: ubuntu-22.04
+    timeout-minutes: 10
+    outputs:
+      dirs: ${{ steps.filter.outputs.changes }}
+    steps:
+      - uses: actions/checkout@v4
+      - uses: dorny/paths-filter@v2
+        id: filter
+        with:
+          filters: |
+            docs: docs/**
+  deploy:
+    needs: [changes]
+    if: ${{ !contains(needs.changes.outputs.dirs, '[]') }}
+    runs-on: ubuntu-22.04
+    timeout-minutes: 10
+    concurrency:
+      group: ${{ github.workflow }}-${{ github.ref }}
+    steps:
+      - uses: actions/checkout@v4
+      - name: Setup mdBook
+        uses: peaceiris/actions-mdbook@v1
+        with:
+          mdbook-version: "latest"
+      - name: Build mdbook
+        working-directory: ./docs
+        run: mdbook build
+      - name: Deploy
+        uses: peaceiris/actions-gh-pages@v3
+        if: ${{ github.ref == 'refs/heads/main' }}
+        with:
+          github_token: ${{ secrets.GITHUB_TOKEN }}
+          publish_dir: ./docs/book
```

Comment:
Does this build the docs on `https://containerd.github.io/runwasi/docs/book`?

Reply:
No, from what I understand, what it does is:
- it generates the `website` at that path
- takes the files and commits them to `gh-pages` branch which can be selected as source for a Github Page.

The problem is it will use the same URL: https://containerd.github.io/runwasi
So, we need to somehow merge the benchmarks with the book also

Reply:
@devigned FYI - can we use the `runwasi.dev` website?